dev/PipsPager/PipsPager.cpp

```diff
@@ -516,6 +516,18 @@ void PipsPager::OnNextButtonClicked(const IInspectable& sender, const winrt::Rou
     SelectedPageIndex(SelectedPageIndex() + 1);
 }
 
+void PipsPager::OnGotFocus(const winrt::RoutedEventArgs& args)
+{
+    m_isPointerOver = true;
+    UpdateNavigationButtonVisualStates();
+}
+
+void PipsPager::OnLostFocus(const winrt::RoutedEventArgs& args)
+{
+    m_isPointerOver = false;
```

Comment:
This creates a bug where having the mouse over the button and moving focus outside the control would hide the button. A different way to solve this would be to have an integer that get's incremented when the pointer is over or focus is inside and get's decremented when the pointer leaves. So if either of the condition get's changed (e.g. focus leaving) but the other "button visible condition" is true, the counter would be 1 and still the buttons would be visible.

Reply:
@chingucoding you're right, counter seems like a good solution to this